schema/artist/index.js

```diff
@@ -439,6 +441,32 @@ export const ArtistType = new GraphQLObjectType({
         type: GraphQLBoolean,
         deprecationReason: "Favor `is_`-prefixed boolean attributes",
       },
+      related: {
```

Comment:
Adds a new `related` field, which currently just holds `suggested`, which is an artists connection based on suggestions.